spec/spec_helper.rb

```diff
@@ -16,10 +16,11 @@
 # in spec/support/ and its subdirectories.
 Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |f| require f }
 
-# Requires factories defined in lib/solidus_expedited_exchanges/factories.rb
-require 'solidus_expedited_exchanges/factories'
+# Requires factories defined in solidus_dev_support
```

Comment:
```suggestion
# Requires factories defined in lib/solidus_expedited_exchanges/testing_support/factories.rb
```

Reply:
Fixed in latest push, thanks for the catch!